ibis/backends/sql/__init__.py

```diff
@@ -463,6 +464,30 @@ def _build_insert_query(
         )
         return query
 
+    def _build_insert_template(
+        self, *, table, schema: sch.Schema, columns: bool = False, fmt: str = "?"
+    ) -> str:
```

Comment:
Since all backends use this, would you mind adding a small docstring, mainly clarifying what the args are?

Also, I _think_ `table` is always passed as `sg.table(name, quoted=quoted)` at every call site - it _might_ be cleaner to only pass the `name` instead to avoid this duplication (don't we set `quoted=True` everywhere anyway?). No strong thoughts.

Reply:
Happy to add a docstring.

sqlite uses a table with a specific catalog, so i'll just leave that bit as is.

Reply:
Eh, I can add a `catalog` argument that'll default to `None` in most cases.